lib/svtplay_dl/utils/retriever_multi.py

```diff
@@ -0,0 +1,111 @@
+# ex:ts=4:sw=4:sts=4:et
+# -*- tab-width: 4; c-basic-offset: 4; indent-tabs-mode: nil -*-
+
+import pycurl
+from io import BytesIO
+import time
+from datetime import timedelta
+
+# We should ignore SIGPIPE when using pycurl.NOSIGNAL - see
+# the libcurl tutorial for more info.
+try:
+    import signal
+    from signal import SIGPIPE, SIG_IGN
+except ImportError:
+    pass
+else:
+    signal.signal(SIGPIPE, SIG_IGN)
+
+from svtplay_dl.utils.output import progressbar, progress_stream, ETA, output
+
+FIREFOX_UA = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.101 Safari/537.3'
```

Comment:
we should import this from utils/http.py

Reply:
Done